src/api/app/views/webui/shared/_download_repository_link.html.haml

```diff
@@ -1,5 +1,10 @@
 - url = "#{download_url}/#{project.to_s.gsub(/:/, ':/')}/#{repository}"
-%li.list-inline-item
-  = link_to(url, title: 'Go to download repository', class: 'nav-link') do
-    %i.fas.fa-download
-    Go to download repository
+- publish_disabled = project.flags.find_by_flag_and_status('publish', 'disable').present?
```

Comment:
And what if a more specific flag on any package in this project overwrites this?

Reply:
I've been thinking into this. The only use case where the download could be completely disabled by the packages is that all packages have the publishing disabled. Because if only one package has publishing enabled, the download page will be present. So I'll loop over the project's packages and mark as `publish_disabled` if the project publishing is disabled or if all the project's packages have the publishing disabled.